libfmt 6.0.0 breaks Waybar

Building Waybar master with libfmt 6.0.0 fails with the following error:
```
126 In file included from ../waybar-9999/include/factory.hpp:4,
127                  from ../waybar-9999/src/factory.cpp:1:
128 ../waybar-9999/include/modules/clock.hpp:5:10: fatal error: fmt/time.h: No such file or directory
129     5 | #include "fmt/time.h"
130       |          ^~~~~~~~
```

Full build log attached: [gui-apps:waybar-9999:20190826-191629.log](https://github.com/Alexays/Waybar/files/3542588/gui-apps.waybar-9999.20190826-191629.log)

--- a/include/modules/clock.hpp
+++ b/include/modules/clock.hpp
@@ -1,8 +1,12 @@
 #pragma once
 
 #include <fmt/format.h>
+#if FMT_VERSION < 60000
+#include <fmt/time.h>
+#else
+#include <fmt/chrono.h>
+#endif
 #include "ALabel.hpp"
-#include "fmt/time.h"
 #include "util/sleeper_thread.hpp"
 
 namespace waybar::modules {

--- a/src/bar.cpp
+++ b/src/bar.cpp
@@ -1,7 +1,7 @@
 #include "bar.hpp"
-#include <spdlog/spdlog.h>
 #include "client.hpp"
 #include "factory.hpp"
+#include <spdlog/spdlog.h>
 
 waybar::Bar::Bar(struct waybar_output* w_output, const Json::Value& w_config)
     : output(w_output),

--- a/src/modules/sni/host.cpp
+++ b/src/modules/sni/host.cpp
@@ -1,6 +1,6 @@
 #include "modules/sni/host.hpp"
-#include <spdlog/spdlog.h>
 #include <fmt/ostream.h>
+#include <spdlog/spdlog.h>
 
 namespace waybar::modules::SNI {
 

